### PR TITLE
test(table): integrated and added test cases for component 

### PR DESCRIPTION
**Pull request by mJarsater**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -40,6 +40,10 @@ function App() {
         <tds-header-item selected={pathname.includes("form")}>
           <Link to="form">Form</Link>
         </tds-header-item>
+        
+        <tds-header-item selected={pathname === '/table'}>
+          <Link to="table">Table</Link>
+        </tds-header-item>
```
Comment (theJohnnyMe) [inline src/App.tsx:46]: Nice!

File: src/components/Table/PaginationTable.tsx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+            country: "Spain",
+            mileage: "1789357",
+        },
+    ]);
+
+    const updateData = (event: any) => {
+        if (tableFooter.current) {
+            console.log(tableFooter.current.getAttribute("pagination-value"));
+            console.log(event.detail.paginationValue);
+            if (event.detail.paginationValue === 1) {
```
Comment (theJohnnyMe) [inline src/components/Table/PaginationTable.tsx:47]: Nice example!

**Comment by theJohnnyMe** [comment]
I wonder if it makes sense to add an example where we add content inside of <tds-table-cell> element as it has a default slot - for those that want to pass HTML inside, like a button or something. What do you think?

**Comment by mJarsater** [comment]
> I wonder if it makes sense to add an example where we add content inside of element as it has a default slot - for those that want to pass HTML inside, like a button or something. What do you think?

Yeah that makes sense!

**Review by theJohnnyMe — APPROVED** [review]
LGTM

**Pull request merged**